server/tests/fixtures/model/articleData.js

```diff
@@ -0,0 +1,10 @@
+import uuid4 from "uuid/v4";
+
+export default {
+  title: "The book of Eli",
+  //authorId: //required. Should be provided by a Users Table
```

Comment:
Remove this it should be here

Reply:
The reason why this is not here is because authorId has a contraints with Users Id, if authorsId matches no id in Users table, creating an Article will fail since Article.authorId must match at least one User.id.

This leaves the responsibility of acquiring userId to Article spec. Which is expected to be implemented as

```
userId = await User.create(validUser).get('id'); //now we know this id exist in user table
articleFixture = (articleFixtures.....);
articleFixture.authorId = userId // guarentee that a matching userId exist
Article.create(articleFixture) //creating article will not complain about User - Article contraints
```